src/Pages/URLShortener/URLShortener.js

```diff
@@ -312,6 +370,7 @@ export default function URLShortenerPage(props) {
                 })}
```

Comment:
Can we add some css for the different columns to give them a fixed width? (lines 330-333 with the example on how to do it on line 332)